src/components/Topnav.vue

```diff
@@ -11,15 +11,15 @@ const { modalAccountOpen } = useModal();
 const { env, domain } = useDomain();
 const route = useRoute();
 
-const { spaces } = useApp();
+const { explore } = useApp();
 const { login, web3 } = useWeb3();
 
 const loading = ref(false);
 const modalWalletNotice = ref(false);
 
 const space = computed(() => {
   const key = domain || route.params.key;
-  return spaces.value[key] ? spaces.value[key] : false;
+  return explore.value.spaces.find(s => s.id === key) ?? false;
```

Comment:
So it should either return a space or return false? 

I would rather return undefined. You are mixing two data types here.